.circleci/config.yml

```diff
@@ -96,9 +96,6 @@ jobs:
       - restore_cache: *restore_yarn_cache
       - run: *run_yarn_install
       - save_cache: *save_yarn_cache
-      - build_app:
-          template: 'Angular InstantSearch'
```

Comment:
why should it not be tested?

Reply:
The template is using Angular 12, which only supports  node 12
https://github.com/angular/angular-cli/blob/master/package.json#L43